### DISCUSSION TITLE
[plugin] org-mouse.nvim

### DISCUSSION THREAD

**Discussion by chipsenkbeil** [nvim-orgmode/orgmode#717] [Show and tell]
@kristijanhusak I split this off from [org-roam.nvim](https://github.com/chipsenkbeil/org-roam.nvim).

[org-mouse.nvim](https://github.com/chipsenkbeil/org-mouse.nvim) adds the mouse highlighting and click capabilities to org documents that I shared earlier. 😄

**Comment by kristijanhusak** [comment]
Mouse?! That's blasphemy! 😄 

Nice :) I saw you added the tag so it should be visible to others.
